### PR TITLE
Add SCA 2024

### PR DESCRIPTION
**Pull request by SnowNation101**
### Which conference does this PR update?

- SCA 2024

### Related URL

- https://computeranimation.org/

### CODE REVIEW

**Comment by jacklightChen** [comment]
Hi, @SnowNation101. We only record the deadlines for abstract/full submission, please consider removing the other deadlines, thanks~

**Comment by SnowNation101** [comment]
sure

File: conference/CG/sca.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  dblp: sca
+  confs:
+    - year: 2024
+      id: sca24
+      link: https://computeranimation.org/
+      timeline:
+        - deadline: '2024-04-22 23:59:59'
+          comment: 'Submission deadline (papers and posters)'
+      timezone: UTC-0
+      date: August 21-23, 2022
```
Comment (jacklightChen) [inline conference/CG/sca.yml:14]: should be 2024?

Comment (jacklightChen) [reply conference/CG/sca.yml:14]: Also please check the timezone selection~ 
image

**Comment by SnowNation101** [comment]
ddl is really near, but yes

在 2024年4月20日，10:50，Zhihao Chen ***@***.***> 写道：

﻿

@jacklightChen commented on this pull request.

________________________________

In conference/CG/sca.yml<https://github.com/ccfddl/ccf-deadlines/pull/483#discussion_r1573119429>:

> @@ -0,0 +1,15 @@
+- title: SCA
+  description: ACM SIGGRAPH/Eurographics Symposium on Computer Animation
+  sub: CG
+  rank: B
+  dblp: sca
+  confs:
+    - year: 2024
+      id: sca24
+      link: https://computeranimation.org/
+      timeline:
+        - deadline: '2024-04-22 23:59:59'
+          comment: 'Submission deadline (papers and posters)'
+      timezone: UTC-0
+      date: August 21-23, 2022


should be 2024?

**Comment by jacklightChen** [comment]
Ok, and could you please modify the year of the conference date: `August 21-23, 2022`

**Comment by SnowNation101** [comment]
modified 🆗

**Review by jacklightChen — APPROVED** [review]

**Pull request merged**